.vscode/extensions.json

```diff
@@ -4,6 +4,6 @@
     "csstools.postcss",
     "bradlc.vscode-tailwindcss",
     "svelte.svelte-vscode",
-    "silvenon.mdx"
+    "unifiedjs.vscode-mdx"
```

Comment:
Ah, I assume `silvenon.mdx` is now deprecated?

Reply:
Yup, this is the new one